src/get-started/java-binding.md

```diff
@@ -501,11 +501,19 @@ Currently Java Binding includes the following built-in services:
   Time oracle allows user services to access the calendar time supplied by
   validator nodes to the blockchain.
 
-To enable a particular service, include its name in `ejb_app_services.toml`
-configuration file in the EJB App's directory with the following content:
+## Services Definition
```

Comment:
Can we clarify the section name? Services Definition sounds too vague. Maybe "Services Activation" or "Node Configuration". The latter is too broad, as there is way more configuration options than this single file. Do we have, however, a section on the node configuration? If not maybe we shall add it with this sub-section and some reference to the usual three-step configuration process (but we also have extensions to that process, which are documented in App_tutorial)?

Seems unclear and easy to get lost. I'd rename this section and submit an issue to develop a better structure/extend this documentation/update App tutorial.

Reply:
created ECR-3048